mautic/mautic
# Place the .gitignore in the project root, not the webroot

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
-->
| Issue(s) addressed                     |

The problem with placing the gitignore in the web-root is that during setups with the recommended project it will place the gitignore in a folder that is one level too deep. Eg. /var/www/application/docroot is where apache or any webserver reads the files from and the .gitignore will be placed in the docroot folder. While you want to deploy the entire application folder and hence also that folder should be in git. This does not cause any issues for regular mautic setups as the project root and web root are the same there.

Also reordering so that all project roots are at the top.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Run git init
3. Run composer install
4. Verify if the plugins and other folders are still ignored by default
